src/content/persons/map-templatekey-person-tracklist-name-victoria-about-personid-uuid-photo-img-victoria_tääker-removebg-preview-png-label-our-people-role-social-human-surname-tääker-linkedin.md

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
This file shouldn't be in the commit probably. Easiest way to get rid of it would be to git reset head~1 and commit again with only files that you changed